### DISCUSSION TITLE
How to connect cryptocurrency wallet with Discord?

### DISCUSSION THREAD

**Discussion by copyNdpaste** [discordjs/discord.js#7961] [Q&A] [answered]
Hello

I want to connect user's wallet to discord.

The flow is like below.

At first I will create 'wallet' channel. And I type message that includes some link or button to connect meta mask chrome extension.

When an user enter 'wallet' channel then click link or button with user's information(user id)

Then send meta mask public key and user id to my server.

My server use ethers.js to get crypto currency balance.

Finally my server grant level to user by balance.

Could you let me know the way or some materials?

**Comment by codegoat-dev** [comment, marked as answer]
You should ask in the Discord server or on stack overflow or something, this is for discussing library functionality.

**Reply by copyNdpaste** [reply]
ok thank u
